package.json

```diff
@@ -98,5 +101,77 @@
   "lint-staged": {
     "*.ts": "yarn run lint:fix",
     "*.svelte": "yarn run lint:fix"
+  },
+  "exports": {
```

Comment:
Have you added it manually? Looks a little bit weird that we need to define each artifact

Reply:
It was created by the migration tool `npx svelte-migrate package` due to their breaking changes